README.md

```diff
@@ -51,3 +51,13 @@ npm run test
 Unit tests will fail if any tests fail, or if code coverage drops below the enforced thresholds. Generally, ideal code coverage levels (for statements/branches/functions/lines) would be around 80%; however, this repo has been set to
 
 To view detailed coverage reports, run the unit tests and then load `./coverage/index.html` in a web browser. This will allow you to view specific coverage information for any file in the repo.
+
+### Integration testing
+
+This repo uses [Cypress](https://www.cypress.io/) for integration testing.
+
+To run integration tests manually, execute the following:
+
+```bash
+$(npm bin)/cypress run
```

Comment:
Ideally we'd use use the `npm run e2e` script (configured [here](https://github.com/trustthevote/MVP-App-TestHarness/blob/main/package.json#L12)) for these tests. Would you mind updating the [configuration for that target](https://github.com/trustthevote/MVP-App-TestHarness/blob/main/angular.json#L133-L147) so that it runs these Cypress tests instead?

That would also allow us to remove all the Protractor references from this repo as well. Given that that [Protractor will be removed from Angular 15 when it's released at the end of next year](https://github.com/angular/protractor/issues/5502), that puts us ahead of the curve.